st2common/st2common/openapi.yaml

```diff
@@ -0,0 +1,3781 @@
+swagger: '2.0'
+
+info:
+  version: "1.0.0"
+  title: StackStorm API
+
+paths:
+  /api/v1/:
+    get:
+      operationId: st2api.controllers.root:root_controller.index
+      description: General API info.
+      responses:
+        '200':
+          description: General API info.
+          schema:
+            type: object
+            properties:
+              version:
+                type: string
+              docs_url:
+                type: string
+          examples:
+            application/json:
+              ref: 'core.local'
+              # and stuff
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  /api/v1/actions:
+    get:
+      operationId: st2api.controllers.v1.actions:actions_controller.get_all
+      x-permissions: {{ PERMISSION_TYPE.ACTION_LIST }}
+      description: Returns a list of all actions.
+      parameters:
+        - name: exclude_attributes
+          in: query
+          description: List of attributes to exclude
+          type: array
+          items:
+            type: string
+          required: false
+        - name: limit
+          in: query
+          description: Number of actions to get
+          type: integer
+        - name: id
+          in: query
+          description: Action id filter
+          type: array
+          items:
+            type: string
+        - name: name
+          in: query
+          description: Action name filter
+          type: string
+        - name: pack
+          in: query
+          description: Action pack name filter
+          type: string
+      responses:
+        '200':
+          description: List of actions
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Action'
+          examples:
+            application/json:
+              ref: 'core.local'
+              # and stuff
```

Comment:
We *could* try and reuse them. I'm hesitant because essentially, examples and fixtures serves different purposes and it would be a constant struggle to compose them in a way that would fit both.